erc/ERCS/erc-7562.md

```diff
@@ -299,41 +394,41 @@ These attacks are examples provided to describe and rationalise the reason for t
 * Total attack cost
     * Polygon (@150 gwei) - $18 + $1.5 per block ($2700/hour)
     * Mainnet: (@40 gwei) - $51840 + $5040 per block ($1.5M/hour)
-    * Note that the above costs ignore the gas-price increase because the underlying network is also overloaded.
+    * Note that the above costs ignore the gas price increase because the underlying network is also overloaded.
 * Mitigations
-    * A **staked factory:** 
+    * A **staked factory:**
         * Can attempt such an attack once, since after invalidating a single bundle creation its reputation drops and another staked factory has to be used.
```

Comment:
```suggestion
        * Can only run one iteration of this attack, since after invalidating a single bundle creation its reputation drops and another staked factory has to be used.
```